src/signals/incident-management/containers/Dashboard/components/ModuleTitle.test.tsx

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { render, screen } from '@testing-library/react'
+
+import { ModuleTitle } from './ModuleTitle'
+
+describe('ModuleTitle', () => {
+  it('should render correctly', () => {
```

Comment:
why not combine these with a rerender?

Reply:
Not sure why. This way it is pretty clear.